src/commons/utils/EnvironmentUtils.ts

```diff
@@ -1,6 +1,6 @@
 class EnvironmentUtilsSingleton {
   private static logger: EnvironmentUtilsSingleton = new EnvironmentUtilsSingleton();
-  private env: string = 'production';
+  private env: string = process.env.NODE_ENV || 'production';
```

Comment:
Non. ici, on veut pas utiliser le NODE_ENV pour definir l'environment de la plateforme.
il faut distinguer NODE_ENV qui correspond à l'environment pour lequel webpack compile (dev ou prod).

Puis les valeurs d'environment possible de la plateforme (dev, qa, prod).

Je vais reverter le changement dans mon prochain commit

Reply:
NODE_ENV = webpack environment
this.env = Coveo Cloud environment